coredocument/read_acls.go

```diff
@@ -140,15 +141,15 @@ func getRole(key []byte, roles []*coredocumentpb.Role) (*coredocumentpb.Role, er
 	return nil, errors.New("role %d not found", key)
 }
 
-// isAccountInRole returns true if account is in the given role as collaborators.
-func isAccountInRole(role *coredocumentpb.Role, account identity.CentID) bool {
-	for _, id := range role.Collaborators {
+// IsAccountInRole returns true if account is in the given role as collaborators.
+func IsAccountInRole(role *coredocumentpb.Role, account identity.CentID) (int, bool) {
```

Comment:
Wouldn't be more succinct if the return was an `int` only? Return index of role in collabs or -1 if not present. Renaming to `FindAccountInRole`.

Reply:
I was think of a pattern that would look like 

`if _, ok := FindAccountInRole; ok {...}` since the index is ignored in most usages except this one.

Reply:
This was one pattern I saw mostly used since checking index for -1 might not be  intuitive IMO

Reply:
Sounds good!

Reply:
Oh, adapt the comment as well.